src/components/FilterPresetsSelect/utils.ts

```diff
@@ -0,0 +1,5 @@
+export const getSeparatorWidth = (paddingValue: string) =>
+  // Separato should cover the whole width of the container,
+  // as the container has padding we have to add it to the width
+  // because container is move to the left by padding value
```

Comment:
**typo:**
```suggestion
  // Separator should cover the whole width of the container,
  // as the container has padding we have to add it to the width
  // because container is moved to the left by the padding value
```